JSDemos/Demos/DataGrid/CustomEditors/Angular/app/app.component.ts

```diff
@@ -67,14 +67,18 @@ export class AppComponent {
     if (target === 'search' && typeof (filterValue) === 'string') {
       return [(this as any).dataField, 'contains', filterValue];
     }
-    return function (data) {
-      return (data.AssignedEmployee || []).indexOf(filterValue) !== -1;
+    return function (rowData) {
```

Comment:
function declaration, why?

Reply:
This is Angular demo, I just renamed parameter _data_ to _rowData_, so it will be same in all demos. In Vue framework of this demo I use an arrow function.